src/main/webapp/content/scss/global.scss

```diff
@@ -806,18 +806,144 @@ Course Info Bar
     }
 }
 
+/* Bootstrap 5 specific fix for flickering tooltips */
+.tooltip {
+    pointer-events: none;
+}
+
+/* Boostrap 5 Print Styles (https://github.com/coliff/bootstrap-print-css/) */
+@media print {
+    *,
+    *::before,
+    *::after {
+        text-shadow: none !important;
```

Comment:
Normally I would suggest to use more specific selectors rather than using `!important` but since this is print-only, it may be fine.

Reply:
This comes from the "official" unofficial [Bootstrap Printing Styles](https://github.com/coliff/bootstrap-print-css/), which I left unchanged.